src/components/backgroundselector/BackgroundSelectorDirective.js

```diff
@@ -78,6 +74,29 @@ goog.require('ga_topic_service');
             });
             return isValid;
           };
+          var updateDefaultBgOrder = function(bgLayers) {
+            bgLayers = bgLayers ? bgLayers : [];
+            defaultBgOrder = [];
+            bgLayers.forEach(function(bgLayerId) {
```

Comment:
I don't think that's fully correct. All topics have the same background layers defined. But the order is different. The first one is the default background layer per topic. blw has pixelkarte farbe, topic are has pixelkarte grau. So it is used.